dev-docs/publisher-api-reference.md

```diff
@@ -445,6 +446,16 @@ Use this method to get all of the bids that have won their respective auctions b
 
 <hr class="full-rule">
 
+<a name="module_pbjs.getTargeting"></a>
+
+### pbjs.adserver.freewheel.getTargeting() ⇒ Object
+
+Use this method to return data on the IAB subcategories mapping files. Data returned includes file URL and optional values refresh rate (in days), and a key identifier. Useful for [building the video url](http://prebid.org/dev-docs/publisher-api-reference.html#module_pbjs.adServers.dfp.buildVideoUrl). 
```

Comment:
Please refer http://prebid.org/dev-docs/publisher-api-reference.html#module_pbjs.adServers.dfp.buildVideoUrl and update this. Below I have added all the details needed

Description: Use this method to get targeting key value pairs to be sent to ad server.

Sample Call:
```
pbjs.adServers.freewheel.getTargeting({
  codes: ['adUnitcode1],
  callback: function(err, targeting) {
    // pass targeting to player api here
  }
});
```

Argument Reference
1) 
Field: codes
Scope: optional
Type: Array
Description: 
Sample: ['adUnitcode1']

2) 
Field: callback
Scope: required
Type: function
Description: Callback function to execute when targeting data is back.
Sample: function(err, targeting) { // pass targeting to player api here }